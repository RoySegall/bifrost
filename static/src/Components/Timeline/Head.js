import React from "react";
import moment from 'moment';
import {dateFormatWithHour} from '../../Services/consts';


const Head = (props) => {
    return <h2> {props['title']}, starts at: {moment(props['startingDate']).format(dateFormatWithHour)} </h2>
};

export default Head;
