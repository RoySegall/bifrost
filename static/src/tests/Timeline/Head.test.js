import React from 'react';
import renderer from 'react-test-renderer';
import Head from '../../Components/Timeline/Head';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {June2519891030} from '../DatesConstans.test';

configure({adapter: new Adapter()});

// Snapshot of the component.
test("Calling the yes no with true", () => {

    const timeline = {
        title: "Title for testing",
        startingDate: June2519891030,
    };
    const component = renderer.create(<Head {...timeline} />);

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
