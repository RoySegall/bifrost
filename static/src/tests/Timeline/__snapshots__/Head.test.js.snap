// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the output of the timeline head component 1`] = `
<h2>
   
  Title for testing
  , starts at: 
  17-11-2019 06:00
   
</h2>
`;
